SH4RE/SH4RE/ViewListing/ViewListingView.swift

```diff
@@ -35,6 +35,9 @@ struct ViewListingView: View {
     @State private var dates: Set<DateComponents> = []
     
     
+    var myRkManager = RKManager(calendar: Calendar.current, minimumDate: Date(), maximumDate: Date().addingTimeInterval(60*60*24*365), mode: 3)
```

Comment:
not a huuuuge fan of this var name... RK is the initials of the dev, like we GET IT, you made the lib...
Whatever, nbd tho.

Reply:
ya i can change